builder-support/debian/authoritative/debian-buster/pdns-backend-ldap.install

```diff
@@ -1,3 +1,4 @@
-modules/ldapbackend/dnsdomain2.schema etc/ldap/schema/
 usr/bin/zone2ldap usr/bin/
 usr/lib/*/pdns/libldapbackend.so*
+usr/share/doc/pdns/dnsdomain2.schema etc/ldap/schema/
```

Comment:
would it make sense to name this pdns-dnsdomain2.schema?

Reply:
maybe, but then it should be named like that in the tarball already

Reply:
@Habbie opinions?

Reply:
it's not a pdns-specific file; let's keep it as is